script/gen_ca.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Maybe we can add a very small comment or README file just describing the purpose of this script? It may worth mentioning that it overwrites current root CA cert files.

Reply:
@amartinezfayo I have both renamed the script as well as added a comment in the header about its operation.

Reply:
Great!